wildfly-modules/src/main/resources/org/infinispan/commons/main/module.xml

```diff
@@ -6,7 +6,7 @@
     <dependencies>
         <module name="javax.api"/>
         <module name="javax.transaction.api"/>
-        <module name="com.github.ben-manes.caffeine" slot="${slot}" export="true"/>
+        <module name="com.github.ben-manes.caffeine"/>
```

Comment:
thanks, that is the line I was wondering about when I asked on IRC if that was necessary. Apparently not :)